script/validators_update_avatar_url.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# RAILS_ENV=production bundle exec ruby script/validators_get_avatar_url.rb
+require File.expand_path('../config/environment', __dir__)
+require File.expand_path('./app/services/keybase_service.rb')
+# Default URL is at 'https://keybase.io/images/no-photo/placeholder-avatar-180-x-180@2x.png'
+
+# include AgentLogic
+# include KeyBaseService
+
+interrupted = false
+trap('INT') { interrupted = true }
+
+agent = Mechanize.new
+keybase_service = KeyBaseService.new
+
+%w[testnet mainnet].each do |network|
+  Validator.where(
+    ["network = ? and keybase_id != '' AND avatar_url IS NOT NULL", network]
+  ).each do |validator|
```

Comment:
Use Active Record for simple queries like this. It's easier to read than just a raw query.